Pipfile

```diff
@@ -32,6 +31,8 @@ python-dateutil = "*"
 bs4 = "*"
 azure = "==4.0.0"
 azure-mgmt-costmanagement = "==0.1.0"
+uritemplate = "*"
+rest-framework = {editable = true,git = "https://github.com/encode/django-rest-framework.git",ref = "master"}
```

Comment:
Some schema generation code depends on fixes currently only found on cutting-edge djangorestframework. 
I understand if merging to master might be blocked until those changes land in a djangorestframework release.

Reply:
@adberglund @chambridge *poke* just to be sure you're aware of this.